salt/_modules/metalk8s_network.py

```diff
@@ -79,3 +79,26 @@ def get_oidc_service_ip():
     range.
     '''
     return _pick_nth_service_ip(OIDC_ADDRESS_NUMBER)
+
+
+def get_mtu_from_ip(ip):
```

Comment:
IP or CIDR?

Reading the [doc](https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.network.html#salt.modules.network.ifacestartswith), I would say CIDR.

Reply:
Not really the doc is wrong here it's not a cidr but an IP (maybe not full)

(And btw it's a bit bugged if you not specify the full IP because he consider 0 as wildcard)

```
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether fa:16:3e:2b:fd:d5 brd ff:ff:ff:ff:ff:ff
    inet 10.200.2.69/16 brd 10.200.255.255 scope global dynamic eth0
       valid_lft 71176sec preferred_lft 71176sec
    inet6 fe80::f816:3eff:fe2b:fdd5/64 scope link 
       valid_lft forever preferred_lft forever
3: tunl0@NONE: <NOARP,UP,LOWER_UP> mtu 1440 qdisc noqueue state UNKNOWN group default qlen 1000
    link/ipip 0.0.0.0 brd 0.0.0.0
    inet 10.233.132.64/32 brd 10.233.132.64 scope global tunl0
       valid_lft forever preferred_lft forever

[root@bootstrap ~]# salt-call network.ifacestartswith 10.200.0.0/16
local:
[root@bootstrap ~]# salt-call network.ifacestartswith 10.200
local:
    - tunl0
    - eth0
[root@bootstrap ~]# salt-call network.ifacestartswith 10.200.
local:
    - eth0
```